e2e/tests/notebook/reader-copy-to-workspace.spec.ts

```diff
@@ -12,6 +12,9 @@ import {waitWhileLoading} from 'utils/waits-utils';
 import WorkspacesPage from 'app/page/workspaces-page';
 import Modal from 'app/modal/modal';
 
+// Retry one more when fails
```

Comment:
Is there a reason we want to retry this test once, but not others? Make clear why this specific test required this intervention, rather than setting something more global.

Reply:
Reason for the retry is because notebook runtime sometimes fail to create, and issue cannot be reproduced when manually retried.